<launch>

   <node pkg="multi_robot_nav" name="static_localization_client" type="static_localization_client.py" ns="/tb3_1" output="screen"/>

</launch>